aea/helpers/async_utils.py

```diff
@@ -412,7 +412,7 @@ def start(self) -> bool:
             self._thread = Thread(
                 target=self._thread_target, name=self.__class__.__name__  # type: ignore # loop was set in set_loop
             )
-            self._thread.setDaemon(True)
+            self._thread.daemon = True
```

Comment:
Was this an api change?

Reply:
A deprecation: https://github.com/valory-xyz/open-aea/pull/371#discussion_r995461494